public/New-DbaAgentSchedule.ps1

```diff
@@ -99,6 +99,9 @@ function New-DbaAgentSchedule {
 
         If force is used the start time will be '23:59:59'
 
+    .PARAMETER OwnerLoginName
```

Comment:
Most Owner parameter are named "Owner":

![image](https://github.com/dataplat/dbatools/assets/66946165/4be88de8-4503-421e-a6c0-bc29379c90fb)

Only Set-DbaAgentJob uses OwnerLogin, none uses OwnerLoginName. I would vote for "Owner" as the parameter name.

And you need to add the new parameter also to the test file.

Reply:
@andreasjordan 

> And you need to add the new parameter also to the test file. 

There are several test that directly or indirectly test New-DbaAgentSchedule, do you mean adding specific test coverage for the Owner parameter, or just populating it in one of the existing tests so it gets indirectly tested?

Reply:
You need to add the new parameter to this list (between EndTime and Force) - or this test will fail: 
https://github.com/dataplat/dbatools/blob/78c1bc392985541999949f76d2738b316e41a534/tests/New-DbaAgentSchedule.Tests.ps1#L9

You could also add a new test that uses the parameter. But I think that is not needed.

After you have changed the test file, I will activate the tests for your PR.